_includes/v20.1/misc/tooling.md

```diff
@@ -0,0 +1,51 @@
+## Support
+
+We’ve partnered with open-source projects, vendors, and individuals to offer the following levels of support with third-party tools. If you encounter problems, please [open an issue](https://github.com/cockroachdb/cockroach/issues/new) with details to help us make progress toward better support.
```

Comment:
I know for certain tools we point to their repos for issue tracking (Ex. [Flyway](http://cockroach-docs-review.s3-website-us-east-1.amazonaws.com/9813de6273efeec08b2336240095608ecad46999/v20.1/flyway.html#report-issues-with-flyway-and-cockroachdb)). I think that is too noisy here so left the general CockroachDB issue tracker.